docs/api/shallow.md

```diff
@@ -46,10 +46,7 @@ describe('<MyComponent />', () => {
   - `options.context`: (`Object` [optional]): Context to be passed into the component
   - `options.disableLifecycleMethods`: (`Boolean` [optional]): If set to true, `componentDidMount`
 is not called on the component, and `componentDidUpdate` is not called after
-[`setProps`](ShallowWrapper/setProps.md) and [`setContext`](ShallowWrapper/setContext.md). Default to `false`. 
-  - `options.lifecycleExperimental`: (`Boolean` [optional]): If set to true, the entire lifecycle
-(`componentDidMount` and `componentDidUpdate`) of the React component is called. The current default value
-is `false` with enzyme v2, but the next major version will flip the default value to `true`.
+[`setProps`](ShallowWrapper/setProps.md) and [`setContext`](ShallowWrapper/setContext.md). Default to `false`.
```

Comment:
I removed lifecycleExperimental mentions but perhaps you want to add some info related to the fat that now shallow has this behaviour by default.